### DISCUSSION TITLE
Need example how to make seeder for nestjs

### DISCUSSION THREAD

**Discussion by purjayadi** [tada5hi/typeorm-extension#236] [General]
I have read the typeorm-extention documentation but until now I still not understand how to create a seeder, can anyone give me an example to do that?

**Comment by tada5hi** [comment]
Hey, sorry ☹️ for the late response. Do you still need advice/help to create a seeder ?

**Reply by theashishmaurya** [reply]
Hey, yes can you tell us or make a documentation about it?

**Comment by arashgl** [comment]
I'm struggling with too can you please provide a documentation for it?
